CHANGELOG.md

```diff
@@ -1,4 +1,33 @@
 ## 2.15.0 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-6.0.2210**
+- Supported Terraform version: **v0.12.x**
+
+### Features:
+1. New data sources:
+  - **data_source_aviatrix_firewall**
+  - **data_source_vpc_tracker**
+2. Implemented support for option to manage attachment on either **aviatrix_vpn_profile** or **aviatrix_vpn_user**
+3. Implemented support for ``action`` under domain_names for **aviatrix_fqdn**
+4. Implemented support for adding new users with Geo VPN
+5. Implemented support for ``ha_peering_subnet`` option for GCP provider for **aviatrix_gateway**
+6. Implemented support for ``ha_subnet`` option for GCP provider for **aviatrix_spoke_gateway** and **aviatrix_transit_gateway**
```

Comment:
5. Implemented support for specifying ``ha_peering_subnet`` for GCP **aviatrix_gateway**
6. Implemented support for specifying ``ha_subnet`` for GCP **aviatrix_spoke_gateway** and **aviatrix_transit_gateway**